hermes/src/api/types.rs

```diff
@@ -316,3 +324,38 @@ impl TryFrom<PriceUpdate> for PriceFeedsWithUpdateData {
         })
     }
 }
+
+#[derive(Debug, Clone, Serialize, Deserialize, ToSchema)]
+pub struct PriceFeedMetadata {
+    pub id:         PriceIdentifier,
+    // BTreeMap is used to automatically sort the keys to ensure consistent ordering of attributes in the JSON response.
+    // This enhances user experience by providing a predictable structure, avoiding confusion from varying orders in different responses.
+    pub attributes: BTreeMap<String, String>,
+}
+
+#[derive(Debug, Serialize, Deserialize, PartialEq)]
+#[serde(rename_all = "lowercase")]
+pub enum AssetType {
+    Crypto,
+    FX,
+    Equity,
+    Metals,
+    Rates,
+}
+
+impl Display for AssetType {
+    fn fmt(&self, f: &mut Formatter) -> FmtResult {
+        write!(
+            f,
+            "{}",
+            match self {
+                // Match each variant to its string representation
+                AssetType::Crypto => "Crypto",
+                AssetType::FX => "FX",
+                AssetType::Equity => "Equity",
+                AssetType::Metals => "Metals",
+                AssetType::Rates => "Rates",
+            }
+        )
```

Comment:
Minor suggestion but if we're matching the names to the same thing as a string you can get away with:

```
write!(f, "{:?}", self)
```